progress/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package progress assists in displaying human readable progress information.
```

Comment:
nit: can we move this package to under `cmd/dist`?
IIUC this should be only used in `dist` (as a dev tool)